File: program15.php
<html>
    <head>
    <title>Swap Numbers</title>
</head>
<body>
<?php
    echo "\nBefore Swapping:";
         $a= 11;
         $b = 10;
         echo "\n a =" .$a. "\nb = ".$b;

         $a = $a + $b;
         $b = $a - $b;
         $a = $a - $b;

         echo "\nAfter swapping No : "."\na = ".$a. "b =".$b;
    
?>
</body>
</html>

File: program15.c

#include<stdio.h>

int main(void)
{

    int a = 5,b= 10,temp;  // swap the values

    temp = a;
    a = b;
    b= temp;

    printf("a= %d\t, b = %d\t",a,b);

    return 0;
}